src/iso/cloud_init_iso.cpp

```diff
@@ -75,10 +75,21 @@ uint8_t to_u8(uint32_t value)
     return static_cast<uint8_t>(value);
 }
 
+uint32_t to_u32(uint8_t value)
+{
+    return static_cast<uint32_t>(value);
+}
+
 std::array<uint8_t, 8> to_lsb_msb(uint32_t value)
 {
-    return {{to_u8(value), to_u8(value >> 8u), to_u8(value >> 16u), to_u8(value >> 24u), to_u8(value >> 24u),
```

Comment:
This is just a manual formatting to make the code look better, no need to review.